src/main/java/uk/gov/companieshouse/acsp/manage/users/interceptor/AuthorizationAndInternalUserInterceptors.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.companieshouse.acsp.manage.users.interceptor;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Objects;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.acsp.manage.users.service.UsersService;
+import uk.gov.companieshouse.acsp.manage.users.utils.StaticPropertyUtil;
+import uk.gov.companieshouse.api.interceptor.InternalUserInterceptor;
+
+@Component
+public class AuthorizationAndInternalUserInterceptors extends AuthorizationInterceptor {
+
+    private final InternalUserInterceptor internalUserInterceptor;
+
+    public AuthorizationAndInternalUserInterceptors( final UsersService usersService ) {
+        super( usersService );
+        this.internalUserInterceptor = new InternalUserInterceptor( StaticPropertyUtil.APPLICATION_NAMESPACE );
+    }
+
+    @Override
+    public boolean preHandle( HttpServletRequest request, HttpServletResponse response, Object handler ) {
```

Comment:
Why the change?  I preferred the other approach you had which was preferring composition (or aggregation) over inheritance.  Not like it's a big deal, this one is similar to the initial example I sent you this morning, but with a few minor differences, just curious on the decision to change is all.